Editor/Window/ConnectToFleetInput.cs

```diff
@@ -127,7 +127,7 @@ private async Task UpdateFleetMenu()
         {
             if (_stateManager.GameLiftWrapper != null)
             {
-                _fleetsList = await _fleetManager.ListFleetAttributes();
+                _fleetsList = await _fleetManager.ListFleetAttributes() ?? new List<FleetAttributes>();
```

Comment:
If there are no fleets, the api call returns an empty list already. This isn't required.